scan-command/src/test/java/io/ballerina/scan/internal/CheckpanicExpressionAnalyzerTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan.internal;
+
+import io.ballerina.projects.Document;
+import io.ballerina.scan.Issue;
+import io.ballerina.scan.RuleKind;
+import io.ballerina.scan.utils.RuleDescription;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+/**
+ * Checkpanic expression analyzer tests.
+ *
+ * @since 0.1.0
+ */
+public class CheckpanicExpressionAnalyzerTest extends StaticCodeAnalyzerTest {
```

Comment:
I would just say

AvoidCheckpanicTest or something like that which reflects the rule.

Reply:
Alternatively, we could go by rule ID too.